integration_tests/metric_test.py

```diff
@@ -260,3 +261,49 @@ def test_qa_metrics(self):
             2,
             "almost equal",
         )
+
+    @unittest.skip(reason="dependency libs are not installed")
+    def test_sql_em(self):
+        # python -m unittest integration_tests.metric_test.MetricTest.test_sql_em
+        metric = explainaboard.metrics.sql_em_ex.SQLEmConfig(
+            name="SQLEm",
+            db_dir="https://expressai-xlab.s3.amazonaws.com/large_data/database",
```

Comment:
This is considered as a personal storage. We also have a Cloud Storage to store large data related to our software. @pfliu-nlp and @neubig have write access to it.